src/gui/guiFormSpecMenu.cpp

```diff
@@ -821,6 +823,8 @@ void GUIFormSpecMenu::parseItemImage(parserData* data, const std::string &elemen
 
 		GUIItemImage *e = new GUIItemImage(Environment, this, spec.fid,
 				core::rect<s32>(pos, pos + geom), name,	m_font, m_client);
+		auto style = getStyleForElement("item_image", spec.fname);
+		e->setNotClipped(style.getBool(StyleSpec::NOCLIP, false));
```

Comment:
Actually, before #8652 `item_image`s weren't clipped, too.

Reply:
If I'm reading correctly, this was merged before FORMSPEC_VERSION was introduced... we could put a check on version 2 which was released in the same version as that patch, but I'm a little worried that someone might see it as a regression or something.
If another developer concurs then I'll implement it, but for now I'm going to leave this unresolved.

Reply:
The same might apply for `list`s btw.